.github/workflows/deploy.yaml

```
@@ -45,13 +45,46 @@ jobs:
+          CURRENT_COUNT=$(aws ecs describe-services \
+            --cluster ${{ vars.ECS_CLUSTER }} \
+            --services ${{ vars.ECS_SERVICE }} \
+            --query 'services[0].desiredCount' \
+            --output text)
+          
+          echo "CURRENT_COUNT=$CURRENT_COUNT" >> $GITHUB_ENV
+
+      - name: 배포를 위한 서비스 확장
+        run: |
+          if [ "$CURRENT_COUNT" -eq 1 ]; then
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)